### PR TITLE
Auth page

### PR DESCRIPTION
**Pull request by TimaVysotski**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,15 +13,23 @@
...
     "@types/react-test-renderer": "^16.9.2",
-    "react": "16.9.0",
+    "i18n-js": "^3.5.1",
+    "react": "^16.9.0",
+    "react-dom": "^16.12.0",
     "react-native": "0.61.5",
+    "react-native-gesture-handler": "1.5.6",
+    "react-native-reanimated": "1.7.0",
+    "react-navigation": "4.1.1",
+    "styled-components": "^5.0.1",
```
Comment (MeyerPV) [inline package.json]: incremental dependencies

File: src/containers/button/index.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement } from 'react';
+import { GestureResponderEvent, TouchableOpacity, Text } from 'react-native';
+import styled from 'styled-components';
+import colors from '../../theme/colors';
+
+interface IButton {
```
Comment (MeyerPV) [inline src/containers/button/index.tsx]: Naming interfaces with ```I``` it's bad practice in Typescript. You can name this like a ButtonProps

File: src/containers/button/index.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  font-size: 18px;
+`;
+
+export const Button = (({ onPress, text }: IButton): ReactElement => (
+  <TouchebleOpacity onPress={onPress}>
+    <ButtonText style={{ color: colors.link }}>{text}</ButtonText>
+  </TouchebleOpacity>
+));
+
+export const SecondaryButton = (({ onPress, text }: IButton): ReactElement => (
```
Comment (MeyerPV) [inline src/containers/button/index.tsx]: Unnecessary component you can just create property, ```textColor``` with default value

File: src/containers/button/index.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+const ButtonText = styled(Text)`
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+`;
+
+export const Button = (({ onPress, text }: IButton): ReactElement => (
+  <TouchebleOpacity onPress={onPress}>
+    <ButtonText style={{ color: colors.link }}>{text}</ButtonText>
```
Comment (MeyerPV) [inline src/containers/button/index.tsx:41]: You need to added this line for you styled(...)`...` ```color: ${({ color }: InfoTextProps): string => color || colors.textTertiary};``` and you color assigne like ``` <ButtonText color={colors.link}>{text}</ButtonText>

File: src/containers/navigation/index.tsx [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import { createAppContainer, createSwitchNavigator } from 'react-navigation';
+import Auth from '../../screens/Auth';
+
+const AppNavigator = createSwitchNavigator({
+  Auth: { screen: Auth },
```
Comment (MeyerPV) [inline src/containers/navigation/index.tsx]: move Auth to the screen constants ( i mean ```key``` ) and use UPPERCASE for all constants

File: src/containers/tabBar/index.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  padding-bottom: 5px;
+  letter-spacing: 3px;
+  color: ${colors.grey};
+  font-size: 16px;
+`;
+
+export const TabBar = ((): ReactElement => (
+  <StyledTabBarView>
+    <StyledText>{'from'}</StyledText>
+    <StyledText style={{ fontWeight: "bold" }}>{'TIMANBLACKMAN'}</StyledText>
```
Comment (MeyerPV) [inline src/containers/tabBar/index.tsx]: move this to styled component props as i wrote upper

File: src/containers/view/container.tsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  children?: Array<React.ReactElement>;
+}
+
+interface IContent {
+  children?: ReactElement;
+}
+
+const StyledView = styled(View)`
+  flex: 1;
+  justify-content: center;
```
Comment (MeyerPV) [inline src/containers/view/container.tsx]: Move this to props with possibility to change this property

**Review by MeyerPV — CHANGES_REQUESTED** [review]

File: src/config/locales/en.json [hunk 8]
```diff
@@ -0,0 +1,9 @@
+{
+  "appName": "Instagram",
+  "button": {
+    "signIn": "Sign In"
+  },
+  "placeholder": {
+    "account": "Create new account"
```
Comment (esnts) [inline src/config/locales/en.json]: next time please write a friendly name for the properties

File: src/containers/button/index.tsx [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+const TouchebleOpacity = styled(TouchableOpacity)`
+  background-color: white;
+  height: 50px;
+  align-items: center;
+  justify-content: center;
+`;
+
+const StyledButton = styled(TouchableOpacity)`
+  background-color: ${colors.link};
+  height: 50px;
```
Comment (esnts) [inline src/containers/button/index.tsx]: use dynamic sizes. 

`5*defualtSize`

File: src/containers/button/index.tsx [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+  width: 100%;
+  border-radius: 5px;
```
Comment (esnts) [inline src/containers/button/index.tsx]: standardize sizes

File: src/containers/tabBar/index.tsx [hunk 11]
```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement } from 'react';
+import { View, Text } from 'react-native';
+import styled from 'styled-components';
+import colors from '../../theme/colors';
+
+const StyledTabBarView = styled(View)`
+  flex: 1;
```
Comment (esnts) [inline src/containers/tabBar/index.tsx:12]: u no need use width with flex

File: src/containers/tabBar/index.tsx [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+const StyledText = styled(Text)`
+  padding-bottom: 5px;
+  letter-spacing: 3px;
+  color: ${colors.grey};
+  font-size: 16px;
+`;
+
+export const TabBar = ((): ReactElement => (
+  <StyledTabBarView>
+    <StyledText>{'from'}</StyledText>
```
Comment (esnts) [inline src/containers/tabBar/index.tsx]: move to i18n

File: src/containers/tabBar/index.tsx [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+`;
+
+const StyledText = styled(Text)`
+  padding-bottom: 5px;
+  letter-spacing: 3px;
+  color: ${colors.grey};
+  font-size: 16px;
+`;
+
+export const TabBar = ((): ReactElement => (
```
Comment (esnts) [inline src/containers/tabBar/index.tsx:29]: export const TabBar = `(`(): ReactElement => (

**Review by esnts — CHANGES_REQUESTED** [review]

**Pull request merged**